app/src/main/res/values/strings.xml

```diff
@@ -421,14 +421,14 @@
     <string name="html_text_dev_feature_encourage">you are encouraging developers to create &lt;strong&gt;new features&lt;/strong&gt; and to &lt;strong&gt;fix bugs&lt;/strong&gt; according to your remarks.</string>
     <string name="html_text_dev_feature_thanks">Thanks a lot for your contribution.</string>
     <string name="html_text_dev_feature_work_hard">We are working hard to release this feature quickly.</string>
-    <string name="html_text_dev_feature_upgrade">Do not forget to keep your app up to date by installing new versions.</string>
+    <string name="html_text_dev_feature_upgrade">Install any new versions to keep the app up to date.</string>
     <string name="download">Download</string>
     <string name="contribute">Contribute</string>
     <string name="download_attachment">Download %1$s</string>
     <string name="download_initialization">Initializing…</string>
     <string name="download_progression">In progress: %1$d\%%</string>
     <string name="download_finalization">Finalizing…</string>
-    <string name="download_complete">Complete! Tap to open the file.</string>
+    <string name="download_complete">Tap to open the file.</string>
```

Comment:
Why delete the status of the completed download?

Reply:
Because a file that was requested for download is implicitly ready when the user can tap to open it.(?)

Reply:
Not necessarily, I can make the clickable notification to access the file path being downloaded.

Reply:
And that means the string doesn't have to be there?

Reply:
And then the string can be removed?